### PR TITLE
Add an howto to deploy object files without building the Executable

### PR DESCRIPTION
**Pull request by tdelhomenie-microej**

### CODE REVIEW

File: SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.. _sdk_6_howto_deploy_object_file_without_building-executable:
+
+How To Build and Deploy Object Files without Building the Executable
+====================================================================
+
+If you need to produce the object files of the Application, deploy them to the BSP, 
+but delegate the build of the Executable to another build system, 
+call the ``buildExecutable`` task with the Application Option ``deploy.bsp.microejscrip`` set to ``false``::
```
Comment (denisbedin) [inline SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst:8]:
```suggestion
call the ``buildExecutable`` task with the Application Option ``deploy.bsp.microejscript`` set to ``false``::
```

File: SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    ./gradlew buildExecutable -Ddeploy.bsp.microejscrip=false
```
Comment (denisbedin) [inline SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst:10]:
```suggestion
    ./gradlew buildExecutable -Ddeploy.bsp.microejscript=false
```

**Review by denisbedin — CHANGES_REQUESTED** [review]

File: SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst (hunk 1 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst]: Done.

File: SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst (hunk 2 above)
Comment (tdelhomenie-microej) [reply SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst]: Done.

File: SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst (hunk 1 above)
Comment (denisbedin) [reply SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst]: Actually, isn't the property `microej.options.deploy.bsp.microejscript` that must be set now ?

Comment (tdelhomenie-microej) [reply SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst]: I don't know. @bguedas ?
Maybe we should update all the properties at the same time in a dedicated PR?

Comment (bguedas) [reply SDK6UserGuide/howtos/buildAndDeployObjectFiles.rst]: Yes, I think it now needs the prefix, but there is no test about this property :/

And yes, we should have anther PR for all the properties to explain how the prefix works

**Pull request merged**